android/src/com/mapswithme/maps/location/GoogleFusedLocationProvider.java

```diff
@@ -118,6 +117,7 @@ public void onResult(@NonNull LocationSettingsResult locationSettingsResult)
           // Location settings are not satisfied. However, we have no way to fix the settings so we won't show the dialog.
           break;
         }
+        requestLocationUpdates();
```

Comment:
Это на юзере не сказывается особо, просто даже если выключены какие-то настройки геолокации, мы всё равно на старте будем пробовать найти его позицию. А по таймауту вывалимся в NOFOLLOW-NOPOSITION

> On May 12, 2016, at 5:26 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/location/GoogleFusedLocationProvider.java <https://github.com/mapsme/omim/pull/3194#discussion_r63029443>:
> 
> > @@ -118,6 +117,7 @@ public void onResult(@NonNull LocationSettingsResult locationSettingsResult)
> >            // Location settings are not satisfied. However, we have no way to fix the settings so we won't show the dialog.
> >            break;
> >          }
> > +        requestLocationUpdates();
> А как это будет выглядеть для юзера? Синяя белочка?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3194/files/f4087a29a78cbab7882e144ef72c93b2b20bdbc9#r63029443>